[11.0][add][purchase_force_invoiced] (#642)
OCA/purchase-workflow: purchase_force_invoiced/readme/USAGE.rst

#. Create a purchase order and confirm it.
#. Receive the products/services.

> #. Create an vendor bill and reduce the invoiced quantity. The purchase order

Reviewer: ```suggestion
#. Create a vendor bill and reduce the invoiced quantity. The purchase order
```

   invoicing status is 'Waiting Bills'.

> #. Lock the Purchase Order, to change the status of it to 'Done'.

Reviewer: ```suggestion
#. Lock the Purchase Order and change its status to 'Done'.
```

> #. Check the field 'Force Invoiced'. Now the quantities of the PO

Reviewer: the sentence is incomplete
